packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -402,21 +525,37 @@ export async function getTransformationPlan(jobId: string) {
             codeTransformRequestId: response.$response.requestId,
             result: MetadataResult.Pass,
         })
-        const logoAbsolutePath = globals.context.asAbsolutePath(
-            path.join('resources', 'icons', 'aws', 'amazonq', 'transform-landing-page-icon.svg')
-        )
-        const logoBase64 = getImageAsBase64(logoAbsolutePath)
-        let plan = `![Amazon Q Code Transformation](${logoBase64}) \n # Code Transformation Plan by Amazon Q \n\n`
-        plan += CodeWhispererConstants.planIntroductionMessage.replace(
-            'JAVA_VERSION_HERE',
-            transformByQState.getSourceJDKVersion()!
-        )
-        plan += `\n\nExpected total transformation steps: ${response.transformationPlan.transformationSteps.length}\n\n`
-        plan += CodeWhispererConstants.planDisclaimerMessage
-        for (const step of response.transformationPlan.transformationSteps) {
-            plan += `**${step.name}**\n\n- ${step.description}\n\n\n`
+
+        const stepZeroProgressUpdates = response.transformationPlan.transformationSteps[0].progressUpdates
+
+        if (!stepZeroProgressUpdates || stepZeroProgressUpdates.length === 0) {
+            // means backend API response wrong and table data is missing
+            throw new Error('No progress updates found in step 0')
         }
 
+        // gets a mapping between the ID ('name' field) of each progressUpdate (substep) and the associated table
+        const tableMapping = getTableMapping(stepZeroProgressUpdates)
+
+        const jobStatistics = JSON.parse(tableMapping['0']).rows // ID of '0' reserved for job statistics table
+
+        // get logo directly since we only use one logo regardless of color theme
+        const logoIcon = getTransformationIcon('transformLogo')
+
+        const arrowIcon = getTransformationIcon('upArrow')
+
+        let plan = `<style>table {border: 1px solid #424750;}</style>\n\n<a id="top"></a><br><p style="font-size: 24px; color: white;"><img src="${logoIcon}" style="margin-right: 15px; vertical-align: middle;"></img><b>${CodeWhispererConstants.planTitle}</b></p><br>`
```

Comment:
NOT a blocker: For better readability, I Would like to follow up with a more string literal/template approach instead of `plan +=`.